Lib/test/test_pathlib.py

```diff
@@ -21,6 +21,15 @@
 except ImportError:
     grp = pwd = None
 
+try:
+    import pwd
+    all_users = [u.pw_uid for u in pwd.getpwall()]
+except (ImportError, AttributeError):
+    all_users = []
+
+root_in_posix = False
+if hasattr(os, 'geteuid'):
+    root_in_posix = (os.geteuid() == 0)
```

Comment:
```suggestion
try:
    root_in_posix = not os.geteuid()
except AttributeError:
    root_in_posix = False
```

This looks to be a copy-and-paste from `test_os.py`, correct? If so, can you move this to https://github.com/python/cpython/tree/main/Lib/test/support somewhere?

Reply:
Sure I can move it and use it from there as a helper module ...